webapp/src/test/java/com/box/l10n/mojito/service/smartling/ThirdPartyTextUnitMatchingServiceTest.java

```diff
@@ -0,0 +1,144 @@
+package com.box.l10n.mojito.service.smartling;
+
+import com.box.l10n.mojito.entity.ThirdPartyTextUnit;
+import com.box.l10n.mojito.service.assetExtraction.ServiceTestBase;
+import com.box.l10n.mojito.service.tm.TMTestData;
+import com.box.l10n.mojito.service.tm.TMTextUnitRepository;
+import com.box.l10n.mojito.smartling.SmartlingClient;
+import com.box.l10n.mojito.test.TestIdWatcher;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.*;
+
+public class ThirdPartyTextUnitMatchingServiceTest extends ServiceTestBase {
+    @Autowired
+    private ThirdPartyTextUnitMatchingService thirdPartyTextUnitMatchingService = new ThirdPartyTextUnitMatchingService();
```

Comment:
can you remove "private" to be consistent with the rest?

Since `@Autowired` remove ` = new ThirdPartyTextUnitMatchingService(`